app/presenters/metrics_presenter.rb

```diff
@@ -36,6 +37,7 @@ def metric_groups
       metric_groups.reverse! if order == Metrics::Order::Descending
 
       if order_by == Metrics::OrderBy::Name.identifier
+        metric_groups = metric_groups.reverse
```

Comment:
I wonder if it'd be clearer to flip the `@order` variable when the `order_by` parameter is name?